<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dateBoard</class>
 <widget class="QWidget" name="dateBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="yearScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>270</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>268</width>
      <height>293</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="monScrollArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>65</y>
     <width>100</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>98</width>
      <height>293</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="dayScrollArea">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>65</y>
     <width>270</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>268</width>
      <height>293</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="currWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>768</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,0,0,0.4);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#D6DAE4;</string>
   </property>
   <widget class="QPushButton" name="canceBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Adobe Heiti Std;
font-weight: normal;
font-size: 26px;
color: #03040D;
border:0px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sureBtn">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>5</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Adobe Heiti Std;
font-weight: normal;
font-size: 26px;
color: #03040D;
border:0px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="m_bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>368</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#fff;</string>
   </property>
  </widget>
  <widget class="QWidget" name="footerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>768</width>
     <height>128</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0.5);
border-top:1px solid #220E0E;</string>
   </property>
  </widget>
  <widget class="QWidget" name="headerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>768</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0.5);
border-bottom:1px solid #220E0E;</string>
   </property>
  </widget>
  <zorder>m_bg</zorder>
  <zorder>yearScrollArea</zorder>
  <zorder>monScrollArea</zorder>
  <zorder>dayScrollArea</zorder>
  <zorder>currWidget</zorder>
  <zorder>widget</zorder>
  <zorder>footerWidget</zorder>
  <zorder>headerWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
